src/mon/Monitor.h

```diff
@@ -586,7 +586,7 @@ class Monitor : public Dispatcher,
 
 public:
   epoch_t get_epoch();
-  int get_leader() { return leader; }
+  int get_leader() const { return leader; }
   const set<int>& get_quorum() { return quorum; }
```

Comment:
@stiopaa1 could you do this sort of cleanup in a batch. for example, add const in to the should-be-const methods of a class in a commit? to avoid too many merge commit? for example, in this commit we can also mark the `get_quorum()` method as `const`.